doc/users.md

```diff
@@ -0,0 +1,29 @@
+[doc](../README.md) > Users
+
+# Users Management
+
+Users information has been introduced in [schema update 07](../server/db/satnogs-07.psql). There's a new table that holds
+user-id, username, salted SHA256 digest, and user role. Until we add flask-admin (or some other management panel), the
+process of adding new users is manual.
+
+To add a new user:
+
+1. log into the server using ssh
+2. psql satnogs
+3. add entry for new user
```

Comment:
Has user the possibility to change password?

Reply:
Sadly, not yet. I was thinking that since we have only 2 users, this is sufficient for now. I can imagine a profile editing panel, where user can edit his first, last name (we don't have that yet), edit his email, change password and possibly do other things. All of that IMHO is a distraction right now. Implementing such features right now does not add value to the project and is a waste of time right now.

This should be pushed to a separate ticket that sadly will be ignored for a while.

Reply:
Created #69 to not forget about this.